analysis_module/serializers.py

```diff
@@ -66,3 +67,26 @@ class PredictionRequestSerializer(serializers.Serializer):
     authoring_organization = serializers.CharField()
     callback_url = serializers.CharField()
     mock = serializers.BooleanField(default=False)
+
+
+class ExtractionDocumentSerializer(serializers.Serializer):
+    url = serializers.CharField()
```

Comment:
Do you think we need to use the TextField instead to avoid limit of 255 chars? It mayn't be applicable here but the presigned s3 url in the response is quite long.

Reply:
In serializers, charfield is used for both, text limit will be checked while saving the object.